File: Boucles/TP4_3.js
function display(array) {
  let i = array.length - 1;
  let currentElement;
  while (i > (array.length - 11) && i >= 0) {
    currentElement = array[i];
    i = i - 1;
    console.log(currentElement)
  }
};
let test = [12, 13, 14];
display(test);

File: Boucles/TP4_3.py
def display(array):
    i=len(array)-1
    while (i > (len(array) - 11) and i >= 0) :
        currentElement = array[i]
        i = i - 1
        print(currentElement)
    return array

test = [12, 13, 14]
display(test)
